<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmoothTalker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/img/icons/transmit.png</normaloff>:/img/icons/transmit.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="vbox_main" stretch="0,10,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Available Rooms:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_rooms">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>This is a list of rooms available to your loggin in accounts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_join_room">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Join Room</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbl_not_connected">
      <property name="text">
       <string>Not Connected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="le_chat_entry">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_chat_submit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_login"/>
    <addaction name="action_logout"/>
    <addaction name="action_options"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dock_user_list">
   <property name="windowTitle">
    <string>Users In Room</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tbl_users">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Users</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/img/icons/door_out.png</normaloff>:/img/icons/door_out.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_login">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/img/icons/connect.png</normaloff>:/img/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Sign In...</string>
   </property>
  </action>
  <action name="action_logout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/img/icons/disconnect.png</normaloff>:/img/icons/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
  <action name="action_options">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/img/icons/wrench.png</normaloff>:/img/icons/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/img/icons/information.png</normaloff>:/img/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_login</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_logout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>logout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_chat_submit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>submit_message()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>730</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_chat_entry</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>submit_message()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>728</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>737</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_join_room</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>join_room()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_options</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_options_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_about</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_about_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
  <slot>logout()</slot>
  <slot>submit_message()</slot>
  <slot>on_test()</slot>
  <slot>join_room()</slot>
  <slot>on_options_activated()</slot>
  <slot>on_about_activated()</slot>
 </slots>
</ui>
